tests/console/sqlite3.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# Summary: Test sqlite3 package
+# Maintainer: Tina Müller <tina.mueller@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+use Data::Dumper;
+
+
+sub run {
+    select_console 'root-console';
+
+    zypper_call('install sqlite3');
+
+    my $archive = "sqlite3-tests.data";
+    assert_script_run(
+        sprintf "cd; curl -L -v %s/data/sqlite3 > %s && cpio -id < %s && mv data sqlite3 && ls sqlite3",
+        autoinst_url(), $archive, $archive
+    );
+
+    # Test various commands
+    assert_script_run("prove -v sqlite3/run-tests.bash");
```

Comment:
Btw openQA has support to parse and display TAP too ^^, so you can save and upload via parse_extra_log( TAP => $file) iir.

Reply:
Ah, cool! Looking...

Reply:
working, thanks!